vlo-web-app/src/main/java/eu/clarin/cmdi/vlo/wicket/provider/SolrDocumentExpansionPairProvider.java

```diff
@@ -22,59 +22,108 @@
 import eu.clarin.cmdi.vlo.pojo.QueryFacetsSelection;
 import eu.clarin.cmdi.vlo.service.solr.SolrDocumentExpansionList;
 import eu.clarin.cmdi.vlo.service.solr.SolrDocumentExpansionPair;
+import eu.clarin.cmdi.vlo.service.solr.impl.SolrDocumentExpansionPairImpl;
 import eu.clarin.cmdi.vlo.service.solr.SolrDocumentService;
 import eu.clarin.cmdi.vlo.wicket.model.SolrDocumentExpansionPairModel;
 import java.math.BigDecimal;
+import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
+
 import org.apache.wicket.markup.repeater.data.IDataProvider;
 import org.apache.wicket.model.IModel;
 
+import eu.clarin.cmdi.vlo.VloWebSession;
+import org.apache.wicket.protocol.http.request.WebClientInfo;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.slf4j.LoggerFactory;
+
+import com.sun.istack.logging.Logger;
+
+import org.apache.wicket.protocol.http.servlet.ServletWebRequest;
+import eu.clarin.cmdi.vlo.pojo.QueryFacetsSelection;
+import eu.clarin.cmdi.vlo.exposure.models.SearchQuery;
+import eu.clarin.cmdi.vlo.exposure.models.SearchResult;
+import eu.clarin.cmdi.vlo.config.ServletVloConfigFactory;
+import eu.clarin.cmdi.vlo.config.VloConfig;
+
 /**
  *
  * @author twagoo
  */
 public class SolrDocumentExpansionPairProvider implements IDataProvider<SolrDocumentExpansionPair> {
+	private final static org.slf4j.Logger logger = LoggerFactory.getLogger(SolrDocumentExpansionPairProvider.class);
```

Comment:
I will review this after the format has been fixed and I can easily see the changes